### PR TITLE
chore(fxa-auth-server): update copy for verifyLogin email

### PR DESCRIPTION
**Pull request by millsoper**
## Because

- We're updating copy in verification/confirmation emails

## This pull request

- Updates copy for the verifyLogin email template

## Issue that this pull request solves

Closes: # (issue number)

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [x] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Screen Shot 2022-08-11 at 2 21 32 PM
Screen Shot 2022-08-11 at 2 21 38 PM

### CODE REVIEW

File: packages/fxa-auth-server/lib/senders/emails/templates/verifyLogin/en.ftl [hunk 1]
```diff
@@ -1,8 +1,6 @@
...
 #  $clientName (String) - A client the user hasn't signed into before (e.g. Firefox, Sync)
-verifyLogin-title = New sign-in to { $clientName }
-verifyLogin-description = For added security, please confirm this sign-in from the following device:
-# Variables:
-#  $clientName (String) - A client the user hasn't signed into before (e.g. Firefox, Sync)
-verifyLogin-subject = Confirm new sign-in to { $clientName }
-verifyLogin-action = Confirm sign-in
+verifyLogin-title-2 = Did you sign in to { $clientName }?
+verifyLogin-description-2 = Help us keep your account safe by confirming you signed in on:
+verifyLogin-subject-2 = Confirm sign in
```
Comment (chenba) [inline packages/fxa-auth-server/lib/senders/emails/templates/verifyLogin/en.ftl:5]: I know the jira asks for "sign in" but I preemptively agree with @bcolsson that it really should be "sign-in".

File: packages/fxa-auth-server/lib/senders/emails/templates/verifyLogin/en.ftl [hunk 2]
```diff
@@ -1,8 +1,6 @@
...
+verifyLogin-action-2 = Confirm sign in
```
Comment (bcolsson) [inline packages/fxa-auth-server/lib/senders/emails/templates/verifyLogin/en.ftl:6]: @chenba beat me to it, but this line should also be "sign-in".

File: packages/fxa-auth-server/lib/senders/emails/templates/verifyLogin/en.ftl (hunk 1 above)
Comment (millsoper) [reply packages/fxa-auth-server/lib/senders/emails/templates/verifyLogin/en.ftl]: Updated

File: packages/fxa-auth-server/lib/senders/emails/templates/verifyLogin/en.ftl (hunk 2 above)
Comment (millsoper) [reply packages/fxa-auth-server/lib/senders/emails/templates/verifyLogin/en.ftl]: Updated

File: packages/fxa-auth-server/lib/senders/emails/templates/verifyLogin/index.txt [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-verifyLogin-description = "For added security, please confirm this sign-in from the following device:"
+verifyLogin-description-2 = "Help us keep your account safe by confirming you signed in on:"
 
 <%- include('/partials/userInfo/index.txt') %>
 
-verifyLogin-action = "Confirm sign-in:"
+verifyLogin-action = "Confirm sign-in"
 <%- link %>
 
 <%- include('/partials/changePassword/index.txt') %>
```
Comment (xlisachan) [inline packages/fxa-auth-server/lib/senders/emails/templates/verifyLogin/index.txt:10]: Please replace `changePassword` partial with `automatedEmailNoAction`, as MJML and TXT currently do not match. Thanks!

Screen Shot 2022-08-15 at 11 49 48 AM

**Review by xlisachan — APPROVED** [review]
LGTM :)

**Review by bcolsson — APPROVED** [review]

**Pull request merged**